projects/core/src/error-handling/effects-error-handler/effects-error-handler.service.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandler, Injectable, inject } from '@angular/core';
+import { Action } from '@ngrx/store';
+import { ErrorAction, HttpErrorModel } from '../../model/index';
+import { WindowRef } from '../../window';
+
+@Injectable()
+export class EffectsErrorHandlerService {
```

Comment:
[MEDIUM]
1. `EffectsErrorHandlerService` vs `ErrorActionService`?
2. if `ErrorActionService`, then what about simply `handle()` instead of `handleError()`?

Reply:
Nice! Like it a lot.